Integrations/GitHub/GitHub.py

```diff
@@ -494,197 +523,84 @@ def format_comment_outputs(comment: dict, issue_number: Union[int, str]) -> dict
     return ec_object
 
 
-''' REQUESTS FUNCTIONS '''
-
-
-def list_teams(organization: str) -> list:
-    suffix = f'/orgs/{organization}/teams'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
-
-def get_branch(branch: str) -> dict:
-    suffix = USER_SUFFIX + f'/branches/{branch}'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
-
-def create_branch(name: str, sha: str) -> dict:
-    suffix = USER_SUFFIX + '/git/refs'
-    data = {
-        'ref': f'refs/heads/{name}',
-        'sha': sha
-    }
-    response = http_request('POST', url_suffix=suffix, data=data)
-    return response
-
-
-def delete_branch(name: str):
-    suffix = USER_SUFFIX + f'/git/refs/heads/{name}'
-    http_request('DELETE', url_suffix=suffix)
-
-
-def get_team_membership(team_id: Union[int, str], user_name: str) -> dict:
-    suffix = f'/teams/{team_id}/memberships/{user_name}'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
-
-def request_review(pull_number: Union[int, str], reviewers: list) -> dict:
-    """Make an API call to GitHub to request reviews from a list of users for a given PR
-
-    Args:
-        pull_number (int): The number of the PR for which the review request(s) is/are being made
-        reviewers (list): The list of GitHub usernames from which you wish to request a review
+''' COMMANDS '''
 
-    Returns:
-        dict: API response
 
-    Raises:
-        Exception: An exception will be raised if one or more of the requested reviewers is not
-            a collaborator of the repo and therefore the API call returns a 'Status: 422 Unprocessable Entity'
-    """
-    suffix = PULLS_SUFFIX + f'/{pull_number}/requested_reviewers'
-    response = http_request('POST', url_suffix=suffix, data={'reviewers': reviewers})
-    return response
+def test_module():
+    http_request(method='GET', url_suffix=ISSUE_SUFFIX, params={'state': 'all'})
+    demisto.results("ok")
 
 
-def create_comment(issue_number: Union[int, str], msg: str) -> dict:
-    suffix = ISSUE_SUFFIX + f'/{issue_number}/comments'
-    response = http_request('POST', url_suffix=suffix, data={'body': msg})
+def create_pull_request(create_vals: dict = {}) -> dict:
+    suffix = PULLS_SUFFIX
+    response = http_request('POST', url_suffix=suffix, data=create_vals)
     return response
 
 
-def list_issue_comments(issue_number: Union[int, str]) -> list:
-    suffix = ISSUE_SUFFIX + f'/{issue_number}/comments'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
+def create_pull_request_command():
+    args = demisto.args()
+    create_vals = {key: val for key, val in args.items()}
```

Comment:
why do we need a new dict, and not just use `args` ?

Reply:
I was averse to modifying the args themselves - I'd rather make modifications to a copy. But I can do that if you'd like.

Reply:
👍